onnxruntime/test/python/transformers/parity_utilities.py

```diff
@@ -3,14 +3,44 @@
 # Licensed under the MIT License.  See License.txt in the project root for
 # license information.
 # -------------------------------------------------------------------------
-
+import argparse
 import os
 import sys
 
 import numpy
 import torch
 
 
+def parse_arguments(namespace_filter=None):
+
+    parser = argparse.ArgumentParser()
+
+    # useful EPs that don't require the use of optmizer.py
+    parser.add_argument(
+        "-n",
+        "--no_optimize",
```

Comment:
It;s wired to define "no_optimize" and get this variable as `optimize`. I think 'no_optimize=True' means we don't want to do optimize. How about use this one:
`parser.add_argument(
        "-n",
        "--no_optimize",
        required=False,
        action="store_true",
        default=False,
        help="Turn on onnxruntime optimizers as default",
    )` 
and use it on test_parity_gelu.py like `TestLayerNormParity.optimized = not args.no_optimize`

Reply:
With argparse you can define a default state especially if its missing. By omitting the no_optimize flag, optimize is set to True by default. In thevother case if you set --no_optimize, the optimize flag is set to false.